### PR TITLE
simple application using enums

### PR DESCRIPTION
**Pull request by Anast-ziya**

### CODE REVIEW

File: OnOffView/OnOffView/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        
+    override func viewDidLoad() {
+        super.viewDidLoad()
+       
+    }
+
+    @IBAction func onOffButtonWasPressed(_ sender: Any) {
+        switchStatus.toggle()
+        if switchStatus == .off {
+            onOffButton.setImage(UIImage(named: "offBtn")!, for: .normal)
```
Comment (EugeneItechart) [inline OnOffView/OnOffView/ViewController.swift:26]: can we avoid using crash operator?

File: OnOffView/OnOffView/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+//  OnOffView
+//
+//  Created by Anastasia Burak on 1.09.21.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    @IBOutlet weak var onOffLabel: UILabel!
```
Comment (EugeneItechart) [inline OnOffView/OnOffView/ViewController.swift:12]: consider of making outlets private ones

File: OnOffView/OnOffView/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    
+    @IBOutlet weak var onOffButton: UIButton!
+    
+    var switchStatus: SwitchStatus = .off
```
Comment (EugeneItechart) [inline OnOffView/OnOffView/ViewController.swift:16]: should it be a public one?

File: OnOffView/OnOffView/ViewController.swift [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        
+    override func viewDidLoad() {
```
Comment (EugeneItechart) [inline OnOffView/OnOffView/ViewController.swift:18]: empty method declaration: it does nothing, so why is it here?

File: OnOffView/OnOffView/ViewController.swift (hunk 1 above)
Comment (Anast-ziya) [reply OnOffView/OnOffView/ViewController.swift:26]: Of course, i'll change it.
This mistake is like in last project

File: OnOffView/OnOffView/ViewController.swift [hunk 5]
```diff
@@ -9,29 +9,30 @@ import UIKit
 
 class ViewController: UIViewController {
     
-    @IBOutlet weak var onOffLabel: UILabel!
+    @IBOutlet private weak var onOffLabel: UILabel!
     
-    @IBOutlet weak var onOffButton: UIButton!
+    @IBOutlet private weak var onOffButton: UIButton!
     
     var switchStatus: SwitchStatus = .off
```
Comment (EugeneItechart) [inline OnOffView/OnOffView/ViewController.swift:16]: why this one is public var switchStatus: SwitchStatus = .off?

File: OnOffView/OnOffView/ViewController.swift [hunk 6]
```diff
@@ -9,29 +9,30 @@ import UIKit
...
         switchStatus.toggle()
         if switchStatus == .off {
-            onOffButton.setImage(UIImage(named: "offBtn")!, for: .normal)
+            if let image = UIImage(named: "offBtn"){
+            onOffButton.setImage(image, for: .normal)
             view.backgroundColor = #colorLiteral(red: 0.2549019754, green: 0.2745098174, blue: 0.3019607961, alpha: 1)
             onOffLabel.text = "🌚 OFF 🌚"
             onOffLabel.textColor = #colorLiteral(red: 1.0, green: 1.0, blue: 1.0, alpha: 1.0)
+            }
         }else{
```
Comment (EugeneItechart) [inline OnOffView/OnOffView/ViewController.swift:28]: pay attention to spacing: } else {
same is here: UIImage(named: "onBtn"){

**Comment by EugeneItechart** [comment]
Could be merged

**Pull request merged**